engine/g5k_engine.py

```diff
@@ -109,6 +109,10 @@ def get_job(self):
             self._make_reservation()
         else:
             logger.info("Using running oargrid job %s" % style.emph(self.gridjob))
+            
+
+        # Wait for the job to start
+        EX5.wait_oargrid_job_start(self.gridjob)
```

Comment:
I think `wait_oargrid_job_start` does what we want: it sleeps until the start date of the job and resumes sleeping in case it is interrupted by a signal.
I have been using it for years and never noticed an issue with nodes. The loop below this code was actually a stupid workaround only necessary because this call was missing and I think we can remove it.